GA/post.py

```
@@ -0,0 +1,1126 @@
+
+        # Initialize fileHandlingMTFLO class - we use the methods developed there to construct the blade shape
+        # Initialized with random inputs since we will not be generating any outputs from the class. 
+        # We just use the methods from the class to generate the geometry for plotting. 
+        fh = fileHandlingMTFLO("*", 1)  
+
+        # Create plot for each stage:
+        for i in range(len(blading)):
+            # Only compute the plots if the stage is to be optimized
+            if config.OPTIMIZE_STAGE[i]:
+                # Set up figures for 2D and 3D plotting
+                fig2d, ax2d = plt.subplots(figsize=(12, 8))
```

Pylint (3.3.7)

[warning] Unused variable 'fig2d'

(W0612)